tests/ttls

```diff
@@ -118,15 +131,18 @@ run_tests() {
 }
 
 title SECTION "TLS with key in provider"
+PROPQ=""
 run_tests
 title ENDSECTION
 
 title SECTION "Forcing the provider for all server operations"
-#Try again forcing all operations on the token
-#We need to disable digest operations as OpenSSL depends on context duplication working
+# We can not put this into the openssl.cnf directly, as it would be picked up by softhsm
+# causing infinite recursion when doing EdDSA key operations.
```

Comment:
Perhaps it is time to ditch softhsm entirely? It seems hopeless and we keep to have to figure out ways to not make it explode...

Reply:
That would be long-shot. I think we will still have to keep it working to some extent, but yes, it pile of trash.